### ISSUE TITLE
Can this be circumvented?

### ISSUE THREAD

**Issue by Lemonsflo** [theabbie/Youtube-Uploader-Bot#6]
On any account asks for confirmation... Can this be fixed somehow?

![fhgfhgf](https://user-images.githubusercontent.com/94681971/143230286-c4b36a7a-0fc1-4a18-9a25-10b2fa61da2f.PNG)

**Comment by theabbie** [comment]
Isn't this a one-time login? It's because you have 2 step verification turned on, you could disable it, but I won't recommend it, it will risk your account security.

**Comment by Lemonsflo** [comment]
No, it's turned off... On any of my accounts it does so... Both with and without 2fa...

**Comment by theabbie** [comment]
https://myaccount.google.com/security

there would be an option to allow less secure apps, you could turn that on, since it's asking for verification on your phone, there must be some option to disable that in your settings, but it will only ask you to log in only once, there's no reason to disable it.
